lib/components/RoundLinkButton/RoundLinkButton.tsx

```diff
@@ -37,7 +37,7 @@ export const RoundLinkButton = ({
     >
       <span
         aria-hidden
-        className={`${selected ? 'text-white' : 'text-black'} material-symbols-outlined size-48 group- flex size-[64px] select-none items-center justify-center self-center rounded-full ${selected ? 'bg-accent' : 'bg-bg-gray hover:text-accent'}`}
+        className={`${selected ? 'text-white' : 'text-black'} material-symbols-outlined size-48 group- flex size-[64px] select-none items-center justify-center self-center rounded-full ${selected ? 'bg-accent' : 'bg-bg-gray-2 hover:text-accent'}`}
```

Comment:
Could these ternaries be combined? `selected ? 'text-white bg-accent' : 'text-black bg-bg-gray'`

Reply:
Yeah, sure. I can do in this ticket while at it.

Reply:
This is now changed. Also noticed the bug with hovering; was able to hove label and the circle shape separately. Fixed to now have only one hover.